README.md

```diff
@@ -42,6 +42,7 @@ README.md file in scripts directory.
 Run the following commands to excute the db migration scripts.
 ```bash
 python manage.py db upgrade
+python manage.py seed 
```

Comment:
The command should be:
```suggestion
python manage.py db seed 
```
because it is nothing but a db action similar to migrate/upgrade.